ocs_ci/templates/provider-client-deployment/ocs_storagecluster.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: ocs.openshift.io/v1
+kind: StorageCluster
+metadata:
+  name: ocs-storagecluster
+  namespace: openshift-storage
+spec:
+  allowRemoteStorageConsumers: true
+  arbiter: {}
+  defaultStorageProfile: ssd-storageprofile
+  encryption:
+    kms: {}
+  externalStorage: {}
+  flexibleScaling: true
+  hostNetwork: true
+  managedResources:
+    cephBlockPools:
+      disableSnapshotClass: true
+      disableStorageClass: true
+      reconcileStrategy: ignore
```

Comment:
we need to drop `reconcileStrategy: ignore`, as per bz https://bugzilla.redhat.com/show_bug.cgi?id=2272401

Reply:
For 4.16 using, ocs_ci/templates/provider-client-deployment/ocs_storagecluster_updated.yaml this yaml for storagecluster it has the latest changes for 4.16.

Reply:
Storageclass validation is part of below function,
verify_storageclient()

Reply:
> Storageclass validation is part of below function,
> verify_storageclient()

sorry, not clear to me.

Reply:
Have updated in latest commit.